_specs/ecip-1107.md

```diff
@@ -0,0 +1,43 @@
+---
+ecip: 1107
+title: WIP - Proof of Work Transition Process Testing
+lang: en
+translator: 
+author: Antoine Toulme (@atoulme)
+discussions-to: https://github.com/ethereumclassic/ECIPs/discussions/444
+status: WIP
```

Comment:
```suggestion
status: Draft
```

Reply:
Interesting, so you think the build is failing due to the WIP designation? How do we submit an ECIP as a WIP then?